resources/components/design/InputForm.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="mb-4">
+    <label class="block mb-2" :for="forValue">{{ label }}</label>
+    <textarea v-if="typeValue === 'textarea'" :value="modelValue" :id="id" :class="inputClass" @input="handleInput"></textarea>
+    <input v-else-if="typeValue === 'file'" :id="id" type="file" accept="image/*" :class="inputClass"
+      @change="handleFileChange" />
+    <input v-else :value="modelValue" :id="id" :type="typeValue" :class="inputClass" @input="handleInput" />
+  </div>
+</template>
+  
+<script setup>
+import { defineProps, defineEmits, computed } from 'vue';
+
+const props = defineProps({
+  label: {
+    type: String,
+    required: false,
+    default: null,
+  },
+  id: {
+    type: String,
+    required: true,
+  },
+  forValue: {
+    type: String,
+    required: true,
+  },
+  typeValue: {
```

Comment:
Pourquoi tu l'appelles pas type au lieu de typeValue ?

Reply:
comme le for faisait bug j'ai aussi modif pour le type

Reply:
met type ça va pas buguer je pense